code/modules/mob/dead/observer/observer.dm

```diff
@@ -903,6 +903,9 @@ GLOBAL_VAR_INIT(observer_default_invisibility, INVISIBILITY_OBSERVER)
 
 	var/choice = tgui_input_list(usr, "You are about to embark to the ghastly walls of Valhalla. Xenomorph or Marine?", "Join Valhalla", list("Xenomorph", "Marine"))
```

Comment:
This is a vital and very related change. Make Marine the first option, is incredibly inconvenient that is the 2nd choice but is used 95% of the time.